cpp/src/hash/hashing.cu

```diff
@@ -38,6 +38,22 @@ bool md5_type_check(data_type dt)
   return !is_chrono(dt) && (is_fixed_width(dt) || (dt.id() == type_id::STRING));
 }
 
+template <typename IterType>
+std::vector<column_view> to_leaf_columns(IterType iter_begin, IterType iter_end)
+{
+  std::vector<column_view> leaf_columns;
+  for_each(iter_begin, iter_end, [&](column_view const& col) {
```

Comment:
Are you intentionally relying on ADL here?

Reply:
No, I updated to specify the capture.